otx/algorithms/classification/adapters/mmcls/models/heads/mixin.py

```diff
@@ -0,0 +1,16 @@
+"""Module defining Mix-in class of heads."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+class OTXHeadMixin:
```

Comment:
Maybe we don't need a unit test for this?

Reply:
I'll add soon.

Reply:
Added in 02e57c70fad225ad826c6ee492f8e46ee87a8e1f